src/hotspot/cpu/riscv/c2_globals_riscv.hpp

```diff
@@ -46,7 +46,7 @@ define_pd_global(intx, OnStackReplacePercentage,     140);
 define_pd_global(intx, ConditionalMoveLimit,         0);
 define_pd_global(intx, FreqInlineSize,               325);
 define_pd_global(intx, MinJumpTableSize,             10);
-define_pd_global(intx, InteriorEntryAlignment,       16);
+define_pd_global(intx, InteriorEntryAlignment,       4);
```

Comment:
It's best to keep InteriorEntryAlignment and CodeEntryAlignment unchanged if there's no obvious benefits.